### ISSUE TITLE
bug with listening to new tracks

### ISSUE THREAD

**Issue by CTAC228** [25huizengek1/ViTune#660]
### Steps to reproduce the bug

It's enough just to try to open a new track that is not cached and the error appears: "The returned video ID doesn't match the requested
one" or "An unknown playback error has occurred"

### Expected behavior

The song should start playing and cached in parallel

### Actual behavior

The song does not play and the above errors occur

### Screenshots/Screen recordings

![Screenshot_20241222_223546](https://github.com/user-attachments/assets/dd0c6cd7-f57f-4278-9fa7-0e39e3f6f56e)
![IMG_20241222_222556_364](https://github.com/user-attachments/assets/ab4ecbac-4ca9-4522-be82-e13761655efe)

### ViTune version

V1.1.1

### What kind of build are you using?

Release (GitHub / F-Droid)

### Android version

Android 13 and Android 15

### Device info

I am using Samsung S20FE 5G on Android 13, and my friend is using Pixel 8 on Android 15

### Checklist

- [x] I am able to reach YouTube Music through the **official** app/website and can confirm that I live in a [supported country](https://support.google.com/youtubemusic/answer/6313540?hl=en) (and this country is not temporarily unavailable)
- [X] I am not using a proxy, VPN, device policy or anything else that restricts access to YouTube Music
- [X] This issue has not been submitted already (this is not a duplicate of another issue)

**Comment by 25huizengek1** [comment]
Duplicate of #505
